addon/components/e3-shape/arc.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  shadowType: 'arc',
+  enterState: {},
+  activeState: {
+    x: 0,
+    y: 0,
+    'start-angle': 0,
+    'angle': 0,
```

Comment:
Well technically you have 2 entries without quotes and 3 with.  It might look better this way:
```
activeState: {
  x: 0,
  y: 0,
  angle: 0,
  'start-angle': 0,
  'inner-radius': 0,
  'outer-radius': null
}
```